apps/nextjs/src/app/(dashboard)/dashboard/(body)/layout.tsx

```
@@ -0,0 +1,17 @@
+interface DashboardBodyLayoutProps {
+  children: React.ReactNode;
+}
+
+export default function DashboardBodyLayout({
+  children,
+}: DashboardBodyLayoutProps) {
+  return (
+    <section
+      role="status"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)